java/client/src/main/java/glide/api/models/commands/LmPopDirection.java

```diff
@@ -0,0 +1,19 @@
+/** Copyright GLIDE-for-Redis Project Contributors - SPDX Identifier: Apache-2.0 */
+package glide.api.models.commands;
+
+import glide.api.commands.ListBaseCommands;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+/**
+ * Enumeration representing element pop direction and count for the {@link ListBaseCommands#lmpop}
+ * command.
+ */
+@RequiredArgsConstructor
+@Getter
+public enum LmPopDirection {
```

Comment:
LEFT and RIGHT enum may be useful for other POP commands.  Maybe just call this `PopDirection`?